docs/tutorial/notebook/train_ss_model_using_pytorch.ipynb

```diff
@@ -0,0 +1,333 @@
+{
```

Comment:
<blockquote><s>Running a test is very similar to training the model. </s></blockquote><p>Instead, say what this will do:</p><p>Next we will evaluate the trained model on the test split.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/isl-org/Open3D-ML/pull/512/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>Done.</p>